### ISSUE TITLE
Grey windows on macOS with yabai/using accesibility feature and qt6.4

### ISSUE THREAD

**Issue by hpfmn** [qutebrowser/qutebrowser#7506]
**Version info**:

```
qutebrowser v2.5.2
Git commit: e56f748cd on qt6-v2 (2022-12-02 18:40:14 +0100)
Backend: QtWebEngine 6.4.1, based on Chromium 102.0.5005.177 (from api)
Qt: 6.4.1 (compiled 6.4.0)

CPython: 3.10.8
PyQt: 6.4.0

sip: no
colorama: 0.4.6
jinja2: 3.1.2
pygments: 2.13.0
yaml: 6.0
adblock: 0.6.0
objc: 9.0
PyQt6.QtWebEngineCore: 6.4.0
pdf.js: 3.1.81 (bundled)
sqlite: 3.39.2
QtNetwork SSL: Secure Transport, macOS 13.0

Style: QMacStyle
Platform plugin: cocoa
OpenGL: None
Platform: macOS-13.0.1-arm64-arm-64bit, 64bit
Frozen: False
Imported from /Users/hpfmn/GIT/qutebrowser/qutebrowser
Using Python from /Users/hpfmn/GIT/qutebrowser/venv/bin/python3.10
Qt library executable path: /Users/hpfmn/GIT/qutebrowser/venv/lib/python3.10/site-packages/PyQt6/Qt6/libexec, data path: /Users/hpfmn/GIT/qutebrowser/venv/lib/python3.10/site-packages/PyQt6/Qt6
OS Version: 13.0.1, arm64
```

**Does the bug happen if you start with `--temp-basedir`?**:

yes

**Description**

use [yabai](https://github.com/koekeishiya/yabai) or probably and other window manager on macOS
open qutebrowser
open a second window

<img width="1440" alt="Screenshot 2022-12-02 at 19 48 20" src="https://user-images.githubusercontent.com/3973466/205365503-f12f7dc0-ec93-42d4-83f3-d6040a47bb6a.png">

<img width="1440" alt="Screenshot 2022-12-02 at 19 49 32" src="https://user-images.githubusercontent.com/3973466/205365521-1e1502fa-ceac-4305-844e-7ea5d687a781.png">

in the console I'm seeing this:

```
❯ ./venv/bin/qutebrowser --temp-basedir
19:47:22 WARNING: Failed to load libssl/libcrypto.
19:47:58 WARNING: skipping QEventPoint(id=1 ts=0 pos=0,0 scn=454.423,512.04 gbl=454.423,512.04 Released ellipse=(1x1 ∡ 0) vel=0,0 press=-454.423,-512.04 last=-454.423,-512.04 Δ 454.423,512.04) : no target window
19:48:14 WARNING: skipping QEventPoint(id=1 ts=0 pos=0,0 scn=848.697,260.82 gbl=848.697,260.82 Released ellipse=(1x1 ∡ 0) vel=0,0 press=-848.697,-260.82 last=-848.697,-260.82 Δ 848.697,260.82) : no target window
19:48:23 WARNING: skipping QEventPoint(id=1 ts=0 pos=0,0 scn=657.037,295.239 gbl=657.037,295.239 Released ellipse=(1x1 ∡ 0) vel=0,0 press=-657.037,-295.239 last=-657.037,-295.239 Δ 657.037,295.239) : no target window
19:48:26 ERROR: At beginning of history.
19:48:43 WARNING: skipping QEventPoint(id=1 ts=0 pos=0,0 scn=791.485,284.052 gbl=791.485,284.052 Released ellipse=(1x1 ∡ 0) vel=0,0 press=-791.485,-284.052 last=-791.485,-284.052 Δ 791.485,284.052) : no target window
19:48:44 WARNING: skipping QEventPoint(id=1 ts=0 pos=0,0 scn=561.012,349.138 gbl=561.012,349.138 Released ellipse=(1x1 ∡ 0) vel=0,0 press=-561.012,-349.138 last=-561.012,-349.138 Δ 561.012,349.138) : no target window
```

**How to reproduce**

see above

**Comment by hpfmn** [comment]
Just confirmed that it also happens with [amethyst](https://github.com/ianyh/Amethyst)

It doesn't happen with qt6.3

**Comment by hpfmn** [comment]
It still there with qt 6.4.1

**Comment by hpfmn** [comment]
Persists with 6.4.2

**Comment by hpfmn** [comment]
Persists with 6.4.3

I'll try if upgrading to macos 13.3 helps

**Comment by hpfmn** [comment]
Persists with macos 13.3 and also on master branch

**Comment by hpfmn** [comment]
I also created a QT Upstream Issue: https://bugreports.qt.io/browse/QTBUG-112385

**Comment by hpfmn** [comment]
But seems to be solved with Qt 6.5!
